### ISSUE TITLE
threshold_py3_staging cannot access PDF documents in Acrobat Reader DC

### ISSUE THREAD

**Issue by lukaszgo1** [nvaccess/nvda#9859]
### Steps to reproduce:
1. Start threshold_py3_staging snapshot of NVDA.
2. Open a PDF file in Acrobat Reader DC.
3. Try to read it.
### Actual behavior:
Nothing is spoken, error sound is played and the following is written in the log
ERROR - eventHandler.executeEvent (15:45:22.921):
error executing event: gainFocus on <NVDAObjects.IAccessible.adobeAcrobat.Document object at 0x05CFA410> with extra args of {}
Traceback (most recent call last):
  File "eventHandler.pyc", line 150, in executeEvent
  File "eventHandler.pyc", line 163, in doPreGainFocus
  File "api.pyc", line 106, in setFocusObject
  File "baseObject.pyc", line 46, in __get__
  File "baseObject.pyc", line 149, in _getPropertyViaCache
  File "NVDAObjects\__init__.pyc", line 504, in _get_container
  File "baseObject.pyc", line 46, in __get__
  File "baseObject.pyc", line 149, in _getPropertyViaCache
  File "NVDAObjects\IAccessible\__init__.pyc", line 968, in _get_parent
  File "NVDAObjects\__init__.pyc", line 127, in __call__
  File "NVDAObjects\IAccessible\adobeAcrobat.pyc", line 61, in initOverlayClass
TypeError: '>' not supported between instances of 'NoneType' and 'int'

### Expected behavior:
It should be possible to interact with Acrobat Reader DC like it is in Python 2 version of NVDA
#### NVDA installed/portable/running from source:
Portable
#### NVDA version:
NVDA version threshold_py3_staging-17927,9812e7b5
#### Windows version:
Windows 7 x64
#### Name and version of other software in use when reproducing the issue:
Acrobat Reader DC Continuous Release | Version 2019.012.20035
#### Does the issue still occur after restarting your PC?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Python 2 Alpha snapshot works as expected.

**Comment by leonardder** [comment]
This is another case of these 'NoneType' to 'int' comparisons.

@feerrenrut: I believe we had several small issues that tried to fix instances of these. 

1. I'd like to know whether your linter picks these up
2. I think we should reconsider this issue and find out whether we could possibly identify more cases of this. Doing all these on a trial and error basis doesn't sound like a good idea to me.

**Comment by feerrenrut** [comment]
> I'd like to know whether your linter picks these up

pyCharm does not, and as far as I can tell, neither flake8 or mypy do either. I tried all 3. It would be quite hard for a linter to pick this up, the value for many of these objects is not easy to predict.

**Comment by feerrenrut** [comment]
Fixed in commit d28b527 via PR #9881
